Bilinear.lua

```diff
@@ -84,6 +84,10 @@ function Bilinear:updateGradInput(input, gradOutput)
    assert(self)
    if self.gradInput then
       self:__assertInputGradOutput(input, gradOutput)
+
+      if #self.gradInput == 0 then
```

Comment:
Sorry about that, I missed the last line when I was copying the fix from somewhere else. I just fixed this.